src/insane/mcmc_cbd.jl

```diff
@@ -637,13 +636,13 @@ function update_λ!(llc    ::Float64,
                    stem   ::Bool,
                    λ_prior::NTuple{2,Float64})
 
-  λp  = randgamma(λ_prior[1] + (ns - 1.0), λ_prior[2] + L)
+  λp  = randgamma(λ_prior[1] + ns, λ_prior[2] + L)
 
   mp  = m_surv_cbd(th, λp, μc, 1_000, stem) 
   llr = log(mp/mc) 
 
   if -randexp() < llr
-    llc += (ns - 1.0) * log(λp/λc) + L * (λc - λp) + llr
+    llc += ns * log(λp/λc) + L * (λc - λp) + llr
```

Comment:
I changed this to account for stem or crown branch. If crown, then is -1 and if stem is -0.

Reply:
Perfect!